app/controllers/candidate_interface/contact_details/address_controller.rb

```diff
@@ -1,5 +1,7 @@
 module CandidateInterface
   class ContactDetails::AddressController < CandidateInterfaceController
+    before_action :redirect_to_dashboard_if_submitted, only: :edit
```

Comment:
Would this allow a intrepid user to update their form by doing a manual POST request to `update`?

Reply:
Possibly, yes @tijmenb but...

```
- When a user has submitted their application
- And they manually navigate to an edit page, such as /candidate/application/personal-details/
- They should be redirected to the dashboard page
```

Reply:
Yeah, I don't think the author of the card considered POST requests, sorry about hat! 

I think we should do all controller actions, because you could have people who have a form open in browser window 1, and the submit in 2, and then press submit in window 1. Their change would still be saved.

Reply:
Yep, POSTs should also be in scope for this, sorry for any confusion!

Reply:
No probs. It makes the code easier so no problem here